### PR TITLE
GLIDER pipeline update:

### PR DESCRIPTION
**Pull request by bpasquer**
- uncomment checks on nraw data in unittest
- set check type for file to be deleted

### CODE REVIEW

File: test_aodndata/anfog/test_anfog.py [hunk 1]
```diff
@@ -256,7 +255,10 @@ def test_deletion_rt_after_dm_upload(self):
         broker.upload(preexisting_files)
 
         # run the handler
-        handler = self.run_handler(GOOD_ZIP_DM)
+
+        handler = self.handler_class(GOOD_ZIP_DM)
```
Comment (lwgordonimos) [inline test_aodndata/anfog/test_anfog.py]:
```python
handler = self.run_handler(GOOD_ZIP_DM, check_params={'checks': ['cf', 'imos:1.4']})
```

File: test_aodndata/anfog/test_anfog.py [hunk 2]
```diff
@@ -304,7 +306,8 @@ def test_renamed_deployment(self):
         broker = get_storage_broker(self.config.pipeline_config['global']['upload_uri'])
         broker.upload(preexisting_files)
 
-        handler = self.run_handler(MISSION_STATUS)
+        handler = self.handler_class(MISSION_STATUS)
```
Comment (lwgordonimos) [inline test_aodndata/anfog/test_anfog.py]: Set this back to run_handler to get the free "no error" assertion!

File: test_aodndata/anfog/test_anfog.py [hunk 3]
```diff
@@ -148,14 +148,13 @@ def test_dstg(self):
 
     def test_nrl(self):
         # test processing of NRL file collection. Collection containn FV01 and FV00
-        # TODO add the NRL FV00 file back in the zip => uncomment last 4 lines
         handler = self.handler_class(ZIP_NRL)
-        # handler.check_params = {'checks': ['cf']}
+        handler.check_params = {'checks': ['cf']}
```
Comment (lwgordonimos) [inline test_aodndata/anfog/test_anfog.py]: See below for how to roll this up into one "run_handler" call

File: test_aodndata/anfog/test_anfog.py [hunk 4]
```diff
@@ -72,7 +72,7 @@ def test_good_dm_file_with_compliance_check(self):
     def test_good_anfog_dm_zip(self):
 
         handler = self.handler_class(GOOD_ZIP_DM)
-        # handler.check_params = {'checks': ['cf', 'imos:1.4']}
+        handler.check_params = {'checks': ['cf', 'imos:1.4']}
```
Comment (lwgordonimos) [inline test_aodndata/anfog/test_anfog.py]: Ditto

File: aodndata/anfog/handlers.py [hunk 5]
```diff
@@ -123,6 +123,8 @@ def process_zip_dm(self):
             if not =>  new deployment, clear RT data
         """
         self.process_zip_common('DM')
+
```
Comment (lwgordonimos) [inline aodndata/anfog/handlers.py]: kill the whitespace!

File: aodndata/anfog/handlers.py [hunk 6]
```diff
@@ -286,6 +288,7 @@ def _set_dm_collection_attributes(self):
 
         # construct collection of files to upload
         upload_to_s3 = self.file_collection.filter_by_attribute_regex('name', AnfogFileClassifier.UPLOAD_TO_S3_REGEX)
+        upload_to_s3.set_check_types(PipelineFileCheckType.FORMAT_CHECK)
```
Comment (lwgordonimos) [inline aodndata/anfog/handlers.py]: Should be redundant, since this is the default

**Comment by bpasquer** [comment]
this last commit is addressing your last comments @lwgordonimos 
please, please, pleeeeease press the merge button once Travis has run his checks

**Pull request merged**